### PR TITLE
pushing up version

### PR DESCRIPTION
**Pull request by foodpoison**
pushing the versions of various things up, and removed build-tools-23.0.1 and 23.0.2, assuming that people will use the most up to date ones within the 23.x versions.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -24,9 +24,9 @@ RUN mv android-sdk-linux /usr/local/android-sdk
 RUN rm android-sdk_r24.4.1-linux.tgz
 
 # Install Android tools
-ENV ANDROID_SUPPORT_VERSION 24.0.0
+ENV ANDROID_SUPPORT_VERSION 24.1.1
 RUN (while :; do echo 'y'; sleep 2; done) | /usr/local/android-sdk/tools/android update sdk --filter extra-android-support,extra-android-m2repository,platform-tool --no-ui -a
-RUN (while :; do echo 'y'; sleep 2; done) | /usr/local/android-sdk/tools/android update sdk --filter build-tools-22.0.1,build-tools-23.0.1,build-tools-23.0.2,build-tools-23.0.3,build-tools-21.1.2,build-tools-24.0.0,build-tools-21.0.3,android-21,android-22,android-15,android-23,android-24 --no-ui -a
+RUN (while :; do echo 'y'; sleep 2; done) | /usr/local/android-sdk/tools/android update sdk --filter build-tools-22.0.1,build-tools-23.0.3,build-tools-21.1.2,build-tools-24.0.1,build-tools-24.0.0,build-tools-21.0.3,android-21,android-22,android-15,android-23,android-24 --no-ui -a
```
Comment (denizmveli) [inline Dockerfile:29]: Maybe a typo? You changed build-tools-23.0.2 to an older version build-tools-21.1.2 ?

Comment (foodpoison) [inline Dockerfile:29]: no no, i removed `build-tools-23.0.1` and `build-tools-23.0.2` assuming that ppl will just keep the latest of 23.0.x which is 23.0.3. The diff is confused at what it's looking at.

Comment (denizmveli) [inline Dockerfile:29]: oh my bad. that didn't change.

**Comment by denizmveli** [comment]
LGTM

**Comment by foodpoison** [comment]
i'm just going to merge now as it takes ages to build these docker images

**Pull request merged**